index.bs

```diff
@@ -1018,6 +1038,32 @@ associated.
         a user's avatar or a [=relying party=]'s logo.
 </div>
 
+
+### Authenticator Selection Criteria ### {#authenticatorSelection}
+
+The {{AuthenticatorSelectionCriteria}} dictionary describes a set of criterias for which the [RPS] can use to find the appropriate
+authenticator 
+
+<xmp class="idl">
+    dictionary AuthenticatorSelectionCriteria {
+        Attachment    attachment = false;
+        boolean       requireResidentKey = false;
+    };
+</xmp>
+
+<div dfn-type="dict-member" dfn-for="AuthenticatorSelectionCriteria">
+    :   <dfn>attachment</dfn>
+    :   If this memeber is [=present|present=], eligible authenticators are filter to only authenticators attached with the 
```

Comment:
Remember to check the rendering: https://api.csswg.org/bikeshed/?url=https://raw.githubusercontent.com/AngeloKai/webauthn/angelo-secondAuthnrSelect/index.bs#dom-authenticatorselectioncriteria-attachment reveals you need two ":"s for this line.